### ISSUE TITLE
Cannot startup fastapi with apm 

### ISSUE THREAD

**Issue by alexcc4** [elastic/apm-agent-python#1184]
With start apm in docker, I just set up agent in fastapi. Then I run app with uvicorn, it hangs.
**To Reproduce**

1. Start Service with docker compose [Reference](https://www.elastic.co/guide/en/apm/get-started/current/quick-start-overview.html)

![image](https://user-images.githubusercontent.com/5210569/124731645-fff5c880-df44-11eb-8b4f-876ec7b0ea3e.png)

- Test apm status
![image](https://user-images.githubusercontent.com/5210569/124731884-3af7fc00-df45-11eb-9f6b-856fc5ef8ae6.png)

2. Run the following simple FastAPI app:
```
from fastapi import FastAPI
from elasticapm.contrib.starlette import make_apm_client, ElasticAPM
import uvicorn


apm = make_apm_client({'SERVICE_NAME': 'test', 'SERVER_URL': 'http://0.0.0.0:8200'})
app = FastAPI()
app.add_middleware(ElasticAPM, client=apm)


@app.get("/")
def read_root():
    return {"Hello": "World"}


if __name__ == '__main__':
    uvicorn.run(app, host='0.0.0.0', port=8000)
```

- Result
![image](https://user-images.githubusercontent.com/5210569/124732134-83171e80-df45-11eb-92ff-bbcc0c3598f4.png)

**Environment**
- OS: Mac
- Python version: `3.7.10`
- Framework and version: `fastapi==0.66.0`
- APM Server version: `image: docker.elastic.co/apm/apm-server:7.13.2`
- Agent version: `elastic-apm==6.3.0`

**Comment by beniwohli** [comment]
Hi @alexcc4! Thanks for opening the issue and providing a test case! I opened a PR with a fix in #1185. We'll make a release with this fix ASAP!

**Comment by beniwohli** [comment]
@alexcc4 version 6.3.1 should fix this issue. Thanks again for the report and sorry about the inconvenience!

**Comment by alexcc4** [comment]
Thanks for quick reply!

**Comment by beniwohli** [comment]
@alexcc4 make sure to update to  6.3.2, the fix in 6.3.1 was faulty :(

**Comment by alexcc4** [comment]
@beniwohli  OK~
